content/en/storage-providers/curio/cli.md

```diff
@@ -0,0 +1,526 @@
+---
+title: "CLI"
+description: "Reference documentation for the curio command-line interface. This documentation was automatically generated using Lotus v1.26.0"
+lead: "Reference documentation for the lotus-miner command-line interface. This documentation was automatically generated using Lotus v1.26.0"
+draft: false
+menu:
+    storage-providers:
+        parent: "curio"
+weight: 145
+toc: true
+---
```

Comment:
This should have alias to the old lotus-provider page so it keeps current links intact.

Reply:
Alias are required to avoid broken links in large docs. There would be no need here as these docs were self contained. I see no link to any of these pages.